solidity/contracts/token/ERC20Token.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity 0.4.26;
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE

```

Comment:
So shall we take this opportunity and migrate to OZ?

Reply:
not yet